tests/utils/profilehandler/data_preparation.go

```diff
@@ -38,28 +41,113 @@ func PreparePrefix(profilePrefix string, nameLength int) string {
 	if nameLength > ocm.MaxClusterNameLength {
 		panic(fmt.Errorf("name length %d is longer than allowed max name length %d", nameLength, ocm.MaxClusterNameLength))
 	}
+	if nameLength == 0 {
+		nameLength = con.DefaultNameLength
+	}
```

Comment:
I didn't get it. When nameLength not set in profile it will be 0.

Reply:
If the profile does not set `nameLength`, use the method `PreparePrefix(profilePrefix, con.DefaultNameLength)` instead of `PreparePrefix(profilePrefix, 0)`.

Reply:
Updated.